hivemind/dht/__init__.py

```diff
@@ -261,18 +261,11 @@ def run_coroutine(
     async def _run_coroutine(
         self, coro: Callable[[DHT, DHTNode], Awaitable[ReturnType]], future: MPFuture[ReturnType]
     ):
-        main_task = asyncio.create_task(coro(self, self._node))
```

Comment:
Cancels here did not work since the new MPFuture implementation does not support asyncio `await`s in a child process.